neurom/check/neuron_checks.py

```diff
@@ -287,3 +287,26 @@ def is_dangling(neurite):
     bad_ids = [(neurite.root_node.id, [neurite.root_node.points[1]])
                for neurite in iter_neurites(n) if is_dangling(neurite)]
     return CheckResult(len(bad_ids) == 0, bad_ids)
+
+
+def has_no_narrow_dendrite(neuron, radius_threshold=0.05, considered_section_min_length=50):
+    '''Check if the neuron has dendrites with narrow sections
+        sections below 'considered_section_min_length' are not taken into account
+    Returns:
+        CheckResult with result. result.info contains the narrow section ids and their
+        first point
+    '''
+    def dendrite_filter(n):
+        '''Select only dendrites'''
+        return n.type == NeuriteType.basal_dendrite or n.type == NeuriteType.apical_dendrite
+
+    considered_sections = filter(lambda sec: sec.length > considered_section_min_length,
+                                 iter_sections(neuron, neurite_filter=dendrite_filter))
+
+    def bad_section(section):
```

Comment:
I would call it narrow_section

Reply:
What about narrow_dendritic_section ?
This makes me think, should the check also be applied to axons as well ?

Reply:
I meant replacing "bad_section" by "narrow_section". Then adding _dendritic_ does not bring much as you are not filtering if it is dendritic in that "bad_section" function.

Reply:
for the axons, I would ask one of the scientists if that is useful before.

Reply:
I see. Renamed.